migrations/01_pools.js

```diff
@@ -13,24 +13,57 @@ module.exports = async (tezos, network) => {
   const factoryAddress = addresses[network]["StakingPoolFactory"];
   const factory = await tezos.contract.at(factoryAddress);
 
-  const depositToken = { token_id: 0, token_address: factoryAddress };
-  const rewardToken = { token_id: 0, token_address: factoryAddress };
+  const dexAddress = "KT1GPJDTf8GZspCcanaG2KhMvGu3NJRqurat";
+  const depositToken_1 = { token_id: 0, token_address: dexAddress };
```

Comment:
Create config for pools configuration, use loop to deploy and display their addresses